modules/L10_system_emulation.sh

```diff
@@ -46,6 +46,8 @@ L10_system_emulation() {
 
     if [[ "${ARCH}" == "MIPS"* || "${ARCH}" == "ARM"* || "${ARCH}" == "x86" ]]; then
 
+      bmc_supermicro
```

Comment:
I would recommend to rename the function to something like "check_bmc_supermicro"